lib/extensions.js

```diff
@@ -470,6 +470,32 @@ module.exports = function(realmConstructor, context) {
                     return "RealmJS/Unknown"
                 }
             },
+
+            _createPlatformDescription() {
+                try {
+                    const sdk_version = require("../package.json").version;
+                    if (typeof navigator !== "undefined" && navigator.product === "ReactNative") {
```

Comment:
could we move this check to a function inside `lib/index.js` and expose it so both here and our `getContext` can use the one and the same function for checking if this environment is RN or node. Otherwise I suspect we will forget to fix this when things change and we will fix only the getContext implementation

Reply:
also I think even better probably call the new function from native code directly instead of having another one in the lib/extensions.js

Reply:
I put it next to the old user agent method but I see your point.

Reply:
I have moved function.